bindgen/lib.rs

```diff
@@ -237,6 +237,7 @@ impl std::fmt::Display for Formatter {
 /// 2. [`bitfield_enum()`](#method.bitfield_enum)
 /// 3. [`newtype_enum()`](#method.newtype_enum)
 /// 4. [`rustified_enum()`](#method.rustified_enum)
+/// 4. [`rustified_non_exhaustive_enum()`](#method.rustified_non_exhaustive_enum)
```

Comment:
I think the Markdown-formatted version will correct "4" to "5" here, but you might care to change this in the raw source also.
```suggestion
/// 5. [`rustified_non_exhaustive_enum()`](#method.rustified_non_exhaustive_enum)
```